index.html

```diff
@@ -1,4 +1,172 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"
+      xmlns="http://www.w3.org/1999/html">
+<head>
+    <link rel=stylesheet href="styles.css">
+    <meta charset="UTF-8">
+    <title>Rekupe</title>
+</head>
+<body>
+<header>
+    <nav class="nav">
+        <ul>
+            <li><a href="#work">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#formulier">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="home">
+        <h1> Your website is an investment in the success of your business.</h1>
+        <h4>We are here to help you succeed.</h4>
+        <button type="button">Let's work together</button>
+    </div>
+    <img id=squiggleone src="assets/squiggle_line_stripe.svg" alt="squiggleone" width="3%">
+</header>
+
 <main>
+    <section class="intro">
+        <div class="one">
+            <h2>Rekupe is a digital agency in Los Angeles.</h2>
+            <h2>We use strategic UX, SEO optimization and beautiful visual designs to help
+                your business grow.</h2>
+            <h2> We see every project as a fresh opportunity, pushing ourselves to create new digital
+                experiences.</h2>
+            <br>
+            <p>Have a question? <a href="#formulier"> Click here</a> to contact us!</p>
+
+        </div>
+    </section>
+
+    <a id="work"></a>
```

Comment:
Hey deze is grappig. Wat doet 'ie daar? Als je de `id="work"` op je section zet heb je hetzelfde resultaat en werken je links nog steeds!